<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinvoiceVelhoValmis</class>
 <widget class="QWidget" name="FinvoiceVelhoValmis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Allekirjoittajalle lähetetään sähköposti, jossa on ohjeet verkkolaskutuksen käyttöön oton vahvistamiseksi.

Verkkolaskutus on käytössä noin tunnin kuluttua allekirjoittamisesta.

Yrityksesi liitetään myös pankkiverkkoon ja verkkolaskuosoitteistoon niin, että voit ottaa vastaan verkkolaskuja kaikkien suomalaisten verkkolaskuoperaattoreiden asiakkailta.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ilmoitus">
     <property name="text">
      <string>Sähköposti-ilmoitus saapuneista laskuista</string>
     </property>
     <property name="icon">
      <iconset resource="../../pic/pic.qrc">
       <normaloff>:/pic/inbox.png</normaloff>:/pic/inbox.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="virhemail">
     <property name="text">
      <string>Sähköposti-ilmoitus virheistä</string>
     </property>
     <property name="icon">
      <iconset resource="../../pic/pic.qrc">
       <normaloff>:/pic/sulje.png</normaloff>:/pic/sulje.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="postiCheck">
     <property name="text">
      <string>Ota käyttöön postitettavien laskujen tulostuspalvelu</string>
     </property>
     <property name="icon">
      <iconset resource="../../pic/pic.qrc">
       <normaloff>:/pic/mail.png</normaloff>:/pic/mail.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
